### DISCUSSION TITLE
GitHub Education Advantages

### DISCUSSION THREAD

**Discussion by Afevkaya** [community/community#130921] [GitHub Education]
### Hi everyone,

Even though the documents were approved on June 30, 2024, the e-mail has not yet arrived and we still cannot benefit from the benefits. Can you help please?

**Comment by Vaneeza-7** [comment]
The same thing happened to a friend of mine. Sometimes, the benefits are available directly in your Github account without an email. Visit the [GitHub Education](https://github.com/education) page and log in to see if the benefits are already available. Also, if your application was approved then the [Student Developer Pack](https://github.com/education/students) should be visible in your account.
